PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,23 @@
+Thank you for contributing to JanusGraph.
+
+In order to streamline the review of the contribution we ask you
+to ensure the following steps have been taken:
+
+### For all changes:
+- [ ] Is there an issue associated with this PR? Is it referenced in the commit message?
+- [ ] Does your PR body contains #xyz where xyz is the issue number you are trying to resolve?
+- [ ] Has your PR been rebased against the latest commit within the target branch (typically ```master```)?
+- [ ] Is your initial contribution a single, squashed commit?
+
+### For code changes:
+- [ ] Have you written and/or updated unit tests to verify your changes?
+- [ ] If adding new dependencies to the code, are these dependencies licensed in a way that is compatible for inclusion under [ASF 2.0](http://www.apache.org/legal/resolved.html#category-a)? 
+- [ ] If applicable, have you updated the LICENSE file, including the main LICENSE file in janusgraph folder?
```

Comment:
By "in janusgraph folder", does this mean "root of this repository"? I understand `git clone` would create the `janusgraph` folder, but that's a user specified name, so it may be ambiguous, but root of the repo probably isn't.

Similarly on the next line.